x/tokenomics/keeper/settle_pending_claims.go

```diff
@@ -168,6 +169,37 @@ func (k Keeper) SettlePendingClaims(ctx sdk.Context) (
 				expiredResult.NumClaims++
 				expiredResult.NumRelays += numClaimRelays
 				expiredResult.NumComputeUnits += numClaimComputeUnits
+
+				// Telemetry - defer telemetry calls so that they reference the final values the relevant variables.
```

Comment:
Can we do a separate function here and below?

I feel like it could even help simplify the code by passing in the claimProofStage